tests/libvmi/vmi.go

```diff
@@ -71,6 +71,13 @@ func WithResourceMemory(value string) Option {
 	}
 }
 
+// WithNamespace specifies the vmi namespace
+func WithNamespace(value string) Option {
```

Comment:
It has been raised that we should not specify the namespace in the spec, it should be left up to the `Create` to add it on creation.

Reply:
Done